torch/_dynamo/utils.py

```diff
@@ -1065,10 +1097,16 @@ def safe_str(item: Any) -> str:
     )
     _compilation_metrics.append(compilation_metrics)
 
-    if compilation_metrics.is_forward:
-        name = "compilation_metrics"
-    else:
-        name = "bwd_compilation_metrics"
+    name = "compilation_metrics"
+    if compilation_metrics.is_forward is False:
+        name = "bwd_" + name
+    if compilation_metrics.is_runtime is True:
+        name = name + "_runtime"
```

Comment:
@ezyang so the new names would be "compilation_metrics_runtime" and "bwd_compilation_metrics_runtime". But I don't know what I need to change to get those to show up in manifold, e.g., https://fburl.com/z3ydyrdm

Reply:
You'll need to update tlparse, I think